setup.cfg

```diff
@@ -1,24 +1,132 @@
+[metadata]
+name = woodwork
+version = 0.14.0
+author = Alteryx Inc.
+license = BSD 3-clause
+long_description = file: READEME.md
+url = https://github.com/alteryx/woodwork/
+long_description_content_type = 'text/markdown'
+author_email = open_source_support@alteryx.com
+keywords = data science machine learning typing
+description = a two-dimensional data object with labeled axes and typing information
+
+classifiers =
+    Development Status :: 3 - Alpha
+    Intended Audience :: Developers
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
+
+project_urls =
+    Documentation = https://woodwork.alteryx.com
+    Source = https://github.com/alteryx/woodwork
+
+[options]
+packages = find:
+python_requires = >=3.7,<4
+include_package_data = True
+
+setup_requires = 
```

Comment:
This section is obsolete and it is not recommended that you have a `setup_requires`.
https://setuptools.pypa.io/en/latest/userguide/dependency_management.html